Src/BootCamp.Chapter/Checks.cs

```diff
@@ -17,25 +17,25 @@ public static class Checks
         public static int PromptInt(string message)
         {
             // To do: call your implementation. 
```

Comment:
|Comments can be removed

Reply:
Removed comments and added second person.